addons/mail/static/tests/core/record.test.js

```diff
@@ -1048,3 +1048,56 @@ test("insert with id relation keeps existing field values", async () => {
     expect(member2.channel.eq(channel1)).toBe(true);
     expect(member2.is_internal).toBe(true);
 });
+
+test("accessing to inverse of a computed field should trigger recomputation", async () => {
+    (class Thread extends Record {
+        static id = "id";
+        id;
+        messageToCompute = Record.one("Message", {
+            compute() {
+                expect.step("compute messageToCompute");
+                return this.messages[0];
+            },
+            inverse: "threadOfComputedMessage",
+        });
+        messages = Record.many("Message");
+    }).register(localRegistry);
+
+    (class Message extends Record {
+        static id = "id";
+        id;
+        thread = Record.one("Thread", { inverse: "messages" });
+        threadOfComputedMessage = Record.one("Thread", { inverse: "messageToCompute" });
+    }).register(localRegistry);
+
+    const store = await start();
+    const thread = store.Thread.insert({ id: 1 });
+
+    let renderCount = 0;
+    function render() {
+        renderCount++;
+        for (const message of reactiveThread.messages) {
+            void message.threadOfComputedMessage;
+        }
+    }
+    const reactiveThread = reactive(thread, render);
+    const message1 = store.Message.insert({ id: 1, thread: thread });
+    expect(
+        `${toRaw(message1)._raw.threadOfComputedMessage.data.map(
+            (localId) => toRaw(thread)._raw.store.get(localId).id
+        )}`
+    ).toBe("1", { message: "threadOfComputedMessage should be computed when inserting message1" });
+    expect(
+        `${toRaw(thread)._raw.messageToCompute.data.map(
+            (localId) => toRaw(thread)._raw.store.get(localId).id
+        )}`
+    ).toBe("1", { message: "messageToCompute should be computed when inserting message1" });
+    expect(renderCount).toBe(1, { message: "render should be called when inserting message1" }); // renderCount is 0 here
+    const message2 = store.Message.insert({ id: 2, thread: thread });
+    expect(
+        `${toRaw(message2)._raw.threadOfComputedMessage.data.map(
+            (localId) => toRaw(thread)._raw.store.get(localId).id
+        )}`
+    ).toBe("1", { message: "threadOfComputedMessage should be computed when inserting message2" });
+    expect(renderCount).toBe(2, { message: "render should be called when inserting message2" }); // renderCount is 0 here
```

Comment:
Although we expect the render function to be called when a new message is inserted, it doesn't happen. So the render count is always `0`. Also `threadOfComputedMessage` in inserting the second message is not set.